File: deber-crud-sails/api/models/Acorde.js
/**
 * Acorde.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {
    notacion_latina:{
      type:'string',
      required:true,
    },
    notacion_inglesa:{
      type:'string',
      required:true,
    },
    img_url:{
      type:'string',
      required:true,
    },
    latitud:{
      type:'number',
      required:true,
    },
    longitud:{
      type:'number',
      required:true,
    },
    url_redireccion:{
      type:'string',
      required:true,
    }


    
       


  },

};

File: Examen1Bimestre/app/src/main/java/com/example/examen1bimestre/Acorde.kt
package com.example.examen1bimestre

class Acorde(
    var id: Int,
    var createdAt: Long,
    var updatedAt: Long,
    var notacion_latina:String,
    var notacion_inglesa:String,
    var img_url:String,
    var latitud:Double,
    var longitud:Double,
    var url_redireccion:String

) {
    override fun toString(): String {
        return "Notaciones-> Latina: ${notacion_latina.capitalize()}, Inglesa:${notacion_inglesa.capitalize()}"
    }

}